test/controllers/user_controller_test.exs

```diff
@@ -33,7 +33,8 @@ defmodule PhoenixTemplate.UserControllerTest do
   end
 
   test "shows chosen resource", %{conn: conn} do
-    user = Repo.insert! %User{}
+    changeset = User.changeset(%User{}, @valid_create_attrs)
```

Comment:
this line is repeated through this file. Are you able to declare the changeset earlier instead of resetting the changeset var in each test?